### ISSUE TITLE
[HELP] Warning: don't forget to edit destination repository to deploy the website?

### ISSUE THREAD

**Issue by cliveb** [gdg-x/hoverboard#22]
In Zeppelin update _config.yml to deploy very clear example: https://github.com/gdg-x/zeppelin/blob/master/_config.yml

In Hoverboard update gulpfile.js to deploy no example (I AM LOST)
https://github.com/gdg-x/hoverboard/blob/master/gulpfile.js#L205

Really appreciate Zeppelin like 
Quick-start guide ;)

Fork this repo
Clone locally
Update _config.yml
Select what content blocks do you need
Push changes to gh-pages branch
Enjoy your awesome DevFest site at http://[your github name].github.io/zeppelin/

**Comment by ozasadnyy** [comment]
Hello @cliveb . Yeah, I know, it's not easy. This is why it's still in beta ;)
For now it will deploy in the same repository you are, but into *gh-pages* branch.
Btw, if you want to deploy into another repository, specify it with **remoteUrl**. Take a look [here](https://github.com/shinnn/gulp-gh-pages#optionsremoteurl). 
If you have enabled two-factors authentication, then take into a count [this issue](https://github.com/shinnn/gulp-gh-pages/issues/72)

**Comment by tasomaniac** [comment]
That's the power of the open source! We should contribute back to the repository with the info we have. 

Zeppelin is really mature. I remember Zeppelin is just like this when it is first released. Hoverboard will be mature and easy to configure too.

**Comment by cliveb** [comment]
Thanks guys. Very much appreciate you making both Zepplin and Hoverboard
for GDG-X. I know documentation is hard (always falls out for sync). A
working sample and where to look is my preference to configure. I'm cooking
up a bootstrap site for DevFest, could not get get even early Zepplin to
instantiate, I have a hunch gh-pages branch needed. I am enjoying the
hacking.

**Comment by ozasadnyy** [comment]
The new version was released. Here is [documentation](https://github.com/gdg-x/hoverboard#deploy-app) for deployment

**Comment by cliveb** [comment]
Hunch Boomerang is a fork of Zeppelin / Hoverboard. Last weekend at gdg #nacommunitysummit I got some mentoring configuring node. I'm using Chromebook + GCP Ubuntu VM to configure (not a Macbook). It turns out on a VM node and npm need manually linking and Ubuntu must be 14.2x. http://gdgseattle.github.io/boomerang/  

You guys may know this Node know-how as background know-how. Is there a way to add this know-how, by adding a Slack channel or some interactive way to get help. I ran into 4 GDGer's at the summit who also had crashed and burned getting github CI hosting working because of Node. For 2015 I ended up forking the BABBQ boostrap to avoid node. https://github.com/GDGSeattle/DevFest

Thanks for consideration.
